### ISSUE TITLE
Warning on unexpected output

### ISSUE THREAD

**Issue by vondele** [cutechess/cutechess#472]

**Comment by ilaripih** [comment]
Yes, if pondering is used then it would be difficult and error-prone to try to automatically determine a reason for the illegal PV move. But without pondering I guess any PV output is unexpected and could be handled with a special error.
